tests/cases/fourslash/inlayHintsShouldWork1.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="fourslash.ts" />
+
+//// function foo (a: number, b: number) {}
+//// foo(/*a*/1, /*b*/2);
+
+const markers = test.markers();
+verify.getInlayHints([
+    {
+        text: 'a:',

```

Comment:
Why do all of the hints include a colon (":")?

Reply:
Not all:
Parameter hints is "name: "
Type hints is ": type"
Enum value hints is "= value"

Reply:
ah, so it's meant to sort of "fill out" the syntax at the call site.

Reply:
Kind of.... I guess